website/docs/general/blog-contributing-guide.md

```diff
@@ -87,6 +112,14 @@ For business users or business cases, you can leave this item blank. Because Git
 
 The author's GitHub avatar, it is related with `authorURL`. For the same reason described  above, this is an optional field. See below for how to get the avatar.
 
+#### authors
+
+Required field for co-author template. When a blog is co-authored by 2 people, please use `authors` instead of `author` to give credits to both authors. `authors` consists of the following fields: `authors.name`, `authors. url`, and `authors.image_url.`
```

Comment:
```suggestion
The required field for co-author template. When a blog is co-authored by 2 people, please use `authors` instead of `author` to give credits to both authors. `authors` consists of the following fields: `authors.name`, `authors.url`, and `authors.image_url.`
```